IDE/Renesas/e2studio/RX72N/EnvisionKit/wolfssl_demo/wolfssl_demo.c

```diff
@@ -455,16 +553,82 @@ static void Tls_client_demo(void)
 
 #endif /* WOLFSSL_RENESAS_TSIP_TLS && (WOLFSSL_RENESAS_TSIP_VER >=109) */
 
+    Tls_client_init();
+
+#ifdef TLS_MULTITHREAD_TEST
+
+    exit_semaph = xSemaphoreCreateCounting(cipherlist_sz, 0);
+
+#ifdef DEBUG_WOLFSSL
+    wolfSSL_SetLoggingCb(my_Logging_cb);
+#endif
+
     do {
-        if(cipherlist_sz > 0 ) printf("cipher : %s\n", cipherlist[i]);
+        for(j = i; j < (i+2); j++) {
+            info[j].id = j;
+            info[j].port = TLSSERVER_PORT + (j%2);
+            info[j].cipher = cipherlist[j];
+            info[j].ctx = client_ctx;
+            info[j].xBinarySemaphore = xSemaphoreCreateBinary();
+            info[j].log_f = my_Logging_cb;
+
+            memset(info[j].name, 0, sizeof(info[j].name));
+            sprintf(info[j].name, "clt_thd_%s", ((j%2) == 0) ?
+                                                            taskA" : "taskB");
+
+            printf(" %s connecting to %d port\n", info[j].name, info[j].port);
+
+            xReturned = xTaskCreate(Tls_client, info[j].name, 
+                                        THREAD_STACK_SIZE, &info[j], 3, NULL);
+            if (xReturned != pdPASS) {
+                printf("Failed to create task\n");
+            }
+        }
+
+        for(j = i; j < (i+2); j++) {
+            xSemaphoreGiveFromISR(info[j].xBinarySemaphore, 
+                                                    &xHigherPriorityTaskWoken);
+        }
+
+        /* check if all tasks are completed */
+        for(j = i; j < (i+2); j++) {
+            if(!xSemaphoreTake(exit_semaph, portMAX_DELAY)) {
+                printf("exit semaphore not released by test task");
+            }
+        }
 
-        Tls_client_init(cipherlist[i]);
+        i+=2;
```

Comment:
Probably need spaces before and after `+=`.

Reply:
Fixed